.gitignore

```diff
@@ -3,4 +3,5 @@
 .idea
 .vagrant
 ubuntu-xenial-core-cloudimg-amd64-root.tar.gz
-.Dockerfile_eventkit.swp
\ No newline at end of file
+.Dockerfile_eventkit.swp
+manifest.yml
```

Comment:
why is `manifest.yml` in the `.gitignore`?  Don't we currently keep it in the repo?